src/planwise/client/projects2/components/settings.cljs

```diff
@@ -95,9 +96,37 @@
   [:div {:class-name "step-header"}
    [:h2 [:span title]]])
 
+
+;-------------------------------------------------------------------------------------------
+; Actions
+(defn- show-action
+  [_ {:keys [idx action-name capacity investment] :as action}]
+  [:div {:class "project-setting"}
+   [m/Button {:type "button"
+              :theme    ["text-secondary-on-secondary-light"]
+              :on-click #(dispatch [:projects2/delete-action action-name idx])}
+    [m/Icon "clear"]]
+   (when (= action-name :build) "with a capacity of ")
+   [current-project-input "" [:config :actions action-name idx :capacity]  "number" {:class "action-input"}]
+   "would cost"
+   [current-project-input "" [:config :actions action-name idx :investment] "number" {:class "action-input"}]])
+
+(defn- listing-actions
+  [action-name list]
+  [:div
+   (for [[index action] (map-indexed vector list)]
+     [show-action {:key (str action-name "-" index)} (assoc action :action-name action-name :idx index)])
+   [m/Button  {:type "button"
+               :theme    ["text-secondary-on-secondary-light"]
+               :on-click #(dispatch [:projects2/create-action action-name])} [m/Icon "add"] "Add Option"]])
+
+;-------------------------------------------------------------------------------------------
+
 (defn current-project-settings-view
   [{:keys [read-only]}]
   (let [current-project (subscribe [:projects2/current-project])
+        build           (subscribe [:projects2/build])
```

Comment:
Name of these subscriptions is confusing. What about `:projects2/build-actions`?